### ISSUE TITLE
v258 Issues

### ISSUE THREAD

**Issue by studlygoorite** [vpinball/vpvr#56]
Just starting new here to try and clean up, the lighting issues are different but still there with some tables.

 Just gave v258 a go and for the first 10 or so tables I saw no issues. Then I ran into lighting issues with Cirqus Volaire, Star Wars Bounty Hunter, Simpsons and Theater Of Magic while Congo was still unplayable with a black screen or green squiggly lines bouncing all over (same as #52) and Demolition Man gave me the dreaded DXGI Error (same as #54) 

Note, I restarted VPVR when I saw issues and tried again. So for me, I see three major issues since 10.7, lighting issues with several tables but many are ok, I can only find one or two tables that just load into a black screen like #52 and lastly the dreaded DXGI Error same as #54 

More testing is needed and I can continue tomorrow but this build is a step in the right direction for me. Thanks

Here is one example of the lighting issue but it is different between tables
![OculusScreenshot1664063628](https://user-images.githubusercontent.com/87738258/192124035-64558d69-23cb-4db6-ab36-632dbb85ad36.jpeg)

**Comment by vbousquet** [comment]
Thanks, this is a good sumup and helped solving bugs.

The latest https://github.com/vpinball/vpvr/actions/runs/3122840547 hopefully should solve most of this. Could you give it a try ?

The only one I was not able to reproduce and work on is https://github.com/vpinball/vpvr/issues/54

**Comment by studlygoorite** [comment]
> Thanks, this is a good sumup and helped solving bugs.
> 
> The latest https://github.com/vpinball/vpvr/actions/runs/3122840547 hopefully should solve most of this. Could you give it a try ?
> 
> The only one I was not able to reproduce and work on is #54

Sure thing

**Comment by studlygoorite** [comment]
> Thanks, this is a good sumup and helped solving bugs.
> 
> The latest https://github.com/vpinball/vpvr/actions/runs/3122840547 hopefully should solve most of this. Could you give it a try ?
> 
> The only one I was not able to reproduce and work on is #54

The lighting issue persists....but Congo now works for me but has the lighting issue. Demo Man still has the #54 issue. I also notice that some tables are now a little darker but were in v258 also like VPW's Simpsons. Not a big issue as I think I could adjust the day/night slider to fix them.

![OculusScreenshot1664125391](https://user-images.githubusercontent.com/87738258/192156134-02b63736-9336-4dd7-8ee5-f70d32164e54.jpeg)
![OculusScreenshot1664124936](https://user-images.githubusercontent.com/87738258/192156157-ab5058a6-8830-4774-a5f2-7692addd0377.jpeg)

![OculusScreenshot1664125211](https://user-images.githubusercontent.com/87738258/192156144-5eadcdce-c36e-4047-a648-08d9d4224024.jpeg)

**Comment by studlygoorite** [comment]
Note, that screen door affect lighting pulsates with the lights on the playfield

**Comment by vbousquet** [comment]
Some questions to help fixing this:
- has demolition man anything special ? a pup pack ? a specific setup ?
- does the lighting artefacts disappear when you turn bloom off ? or another postprocess effect ? Also could you post your settings (video & vr) ?

Edit: I was finally able to reproduce (thanks to your screenshots, with Cirqus Voltaire). It is the Bloom filter when in VR only. Investigating...

**Comment by studlygoorite** [comment]
> Some questions to help fixing this:
> 
> * has demolition man anything special ? a pup pack ? a specific setup ?
> * does the lighting artefacts disappear when you turn bloom off ? or another postprocess effect ? Also could you post your settings (video & vr) ?
> 
> Edit: I was finally able to reproduce (thanks to your screenshots, with Cirqus Voltaire). It is the Bloom filter when in VR only. Investigating...

Sorry I missed your request to try v234 nine days ago as I was away fishing :) Nice catch, Demolition Man does have a pup pack and after removing the pup pack the table started and played just fine.
 I adjusted "bloom strength" to zero and it did not fix the lighting issue with Congo and Star Wars Bounty Hunter. I have asked in the VR Discord if that is the proper way to turn bloom off. My setting are posted in #36 and good news you can finally reproduce the issue with Cirqus, it might have something to do with all of the lighting issues.

**Comment by studlygoorite** [comment]
I'll just throw this out there but Rob Ross wrote this about the DXGI Error  (#54 ) on Facebook/Meta which started quite a few builds back.

DXGI errors mean you are having some problem capturing external displays (PUP/DMD). Could mean you're trying to capture a window that's half off screen, or using a laptop with a hybrid GPU not supported by DXGI.

**Comment by vbousquet** [comment]
I was able to track down the lighting artefact. The latest version here is expected to be fully ok (except pup pack): https://github.com/vpinball/vpvr/actions/runs/3123402510

Could you give it a try ?

**Comment by studlygoorite** [comment]
> I was able to track down the lighting artefact. The latest version here is expected to be fully ok (except pup pack): https://github.com/vpinball/vpvr/actions/runs/3123402510
> 
> Could you give it a try ?

You da man!! No more lighting artifacts. I'm going to do a thorough test of many tables later this afternoon. Thanks for your time.

**Comment by vbousquet** [comment]
Also, regarding the pupack bug. If you know when it started to stop working, it would help me track it down. I don't think I will have the time to look it before next week-end though (I'm in Europe and night has come here).

**Comment by studlygoorite** [comment]
Sure thing. Good night

**Comment by studlygoorite** [comment]
Good news, I just tested 60-ish tables with v266 that all looked and played great, a couple had bulbs or lights that were a little bit too bright but that's it.

300, Addams Family, Apollo 13, Atlantis 1989, Attack From Mars, Avatar (but no pup played), Back To The Future, Bad Cats, Batman DK (senseless I think) not Bord's new one, Big Shot, Black Knight, Black Knight 2000, Blackout, Black Pyramid, Black Rose, Blood Machines, Cactus Jacks, Cactus Canyon, Cheech and Chong, Cirqus Voltaire (bright right ramp), Class of 1812, Comet, Congo, Creature From Black Lagoon, Defender, Demolition Man (without pup), Diner, Dr Who, Elvis, F-14, Family Guy, Fathom, Firepower, Fish Tales, Flintstones, Friday the 13th, Genie, Getaway, Ghostbusters, Judge Dredd, Maverick, Pharaoh, Pinball Magic, Stellar Wars, Stranger Things, Tales From The Crypt, TOTAN, Taxi, Teed Off, Terminator 2 and 3, Twilight Zone, Triple Strike, Twister, Unreal Tournament, Young Frankenstein, Whirlwind and WWF Royal Rumble all were fine.

I only had bad issues with three of the 60-ish tables, AC/DC either the music would not play or the pup pack was all red, another was Jaws, I got the DXGI Error that I had to force close everything with task manager and finally VooDoo's Carnival, the backglass would not show.  (maybe not the fault of the build, could be something on my end) and tables that use pup packs just displayed backglasses but did not crash like Demolition Man.  Attack From Mars, Indiana Jones (offset DMD also) and Avatar just showed the backglass and with Austin Powers the pup topper was way too dark. More testing of tables with pup packs is needed 

Others are going to test the build, if they don't post here I will share the info when it comes. Thanks again!!
